src/rtctools_heat_network/workflows/utils/adapt_profiles.py

```diff
@@ -0,0 +1,150 @@
+import datetime
+import logging
+
+import numpy as np
+
+from rtctools.data.storage import DataStore
+
+logger = logging.getLogger("WarmingUP-MPC")
+logger.setLevel(logging.INFO)
+
+
+def adapt_hourly_year_profile_to_day_averaged_with_hourly_peak_day(problem, problem_day_steps):
+    """
+    Adapt yearly porifle with hourly time steps to a common profile (daily averaged profile except
+    for the day with the peak demand).
+
+    Return the following:
+        - problem_indx_max_peak: index of the maximum of the peak values
+        - heat_demand_nominal: max demand value found for a specific heating demand
+    """
+
+    def set_data_with_averages_and_peak_day(
+        datastore: DataStore,
+        variable_name: str,
+        ensemble_member: int,
+        new_date_times: np.array,
+    ):
```

Comment:
Why is this function inside the adapt... function?

Reply:
Sorry. My bad